File: Homeworks/C# Part 1/03.OperatorsAndExpressions/10.PointCircleRectangle/PointCircleRectangle.cs
using System;

class PointCircleRectangle
{
    static void Main()
    {
        double numberX = double.Parse(Console.ReadLine());
        double numberY = double.Parse(Console.ReadLine());
        double distanceCircle;
        distanceCircle = Math.Sqrt((numberX-1) * (numberX-1) + (numberY-1) * (numberY-1));
        if (distanceCircle > 1.5)
        {
            Console.Write("outside circle ");
        }
        else
        {
            Console.Write("inside circle ");
        }
        if (numberX > 5 || numberX < -1 || numberY > 1 || numberY < -1)
        {
            Console.WriteLine("outside rectangle");
        }
        else
        {
            Console.WriteLine("inside rectangle");
        }
    }
}

File: Homeworks/JS Fundamentals/04. Operators and Expressions/09. Point, Circle, Rectangle/point-circle-rectangle.js
function solve(args) {
	var numberX = +args[0];
    var numberY = +args[1];
    var distanceCircle;
    var stateCircle;
    var stateRectangle;
    distanceCircle = Math.sqrt((numberX-1) * (numberX-1) + (numberY-1) * (numberY-1));
    if (distanceCircle > 1.5)
    {
    stateCircle = 'outside';
    }
    else
    {
    stateCircle = 'inside';
    }
    if (numberX > 5 || numberX < -1 || numberY > 1 || numberY < -1)
    {
    stateRectangle = 'outside';
    }
    else
    {
    stateRectangle = 'inside';
    }
    console.log(stateCircle + ' circle ' + stateRectangle + ' rectangle');
}